lib/cassanity/executors/cql_rb.rb

```diff
@@ -185,7 +184,7 @@ def call(args = {})
             end
 
             # Select the correct keyspace before executing the CQL query
-            if command != :keyspace_create && (keyspace_name = arguments[:keyspace_name])
+            if !command.to_s['keyspace'] && (keyspace_name = arguments[:keyspace_name])
```

Comment:
This one is to not enter the if branch if the command contains the `keyspace` word as we don't want to use any keyspace if operating on them. This is because in the new driver, if the session is using a particular keyspace that doesn't exist fails. Then, I wanted to add `:keyspace_drop` to the if, but then realised that a general keyspace regex would be better. Makes sense?